Formula/makensis.rb

```diff
@@ -46,7 +46,7 @@ def install
     end
 
     # Don't strip, see https://github.com/Homebrew/homebrew/issues/28718
-    scons "STRIP=0", "ZLIB_W32=#{@zlib_path}", "makensis"
+    scons "STRIP=0", "ZLIB_W32=#{@zlib_path}", "SKIPUTILS=NSIS Menu", "makensis"
```

Comment:
The NSIS Menu is basically a welcome screen with links to local and online resources. It's pretty useless on non-Windows, so let's save some build-time. This also matches the [Travis CI settings](https://github.com/kichik/nsis/blob/master/.travis.yml#L16) of the NSIS repository